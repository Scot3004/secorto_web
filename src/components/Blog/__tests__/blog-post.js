import React from "react"
import renderer from "react-test-renderer"
import BlogPost from "../blog-post"

describe("BlogPost", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <BlogPost title="blog-post" image="testImage" date="2021-10-14">
          content
        </BlogPost>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
