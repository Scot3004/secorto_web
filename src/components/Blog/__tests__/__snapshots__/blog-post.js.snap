// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogPost renders correctly 1`] = `
.emotion-0 {
  padding: 1rem;
}

.emotion-1 {
  color: primary;
  margin-bottom: 0;
}

.emotion-2 {
  margin-bottom: 16px;
}

.emotion-3 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
  position: relative;
  top: -2px;
}

<div
  className="emotion-0"
>
  <img
    src="testImage"
  />
  <h1
    className="emotion-1"
  >
    blog-post
  </h1>
  <div
    className="emotion-2"
  >
    <small>
      <svg
        className="emotion-3 emotion-4"
        fill="currentColor"
        focusable="false"
        height={16}
        role="img"
        viewBox="0 0 24 24"
        width={16}
        xmlns="http://www.w3.org/2000/svg"
      >
        <title>
          Fecha
        </title>
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <path
          d="M20 3h-1V1h-2v2H7V1H5v2H4c-1.1 0-2 .9-2 2v16c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zm0 2v3H4V5h16zM4 21V10h16v11H4z"
        />
        <path
          d="M4 5.01h16V8H4z"
          opacity={0.3}
        />
      </svg>
       
      jueves, 14 de octubre de 2021
    </small>
  </div>
  content
</div>
`;
